From b1ad4444ae92156e21972ce74eaafb82e0ac61d7 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Tue, 13 Feb 2024 15:33:37 +0000
Subject: [PATCH 0894/1085] arm: dt: bcm2712: Reduce DDC frequency to 97.5kHz
 from 200kHz.

The I2C spec says the DDC link should run at 100kHz or less, however
Pi5/BCM2712 had been configured for 200kHz.
Reduce it to comply with the spec, and match Pi4.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2712.dtsi
@@ -442,7 +442,7 @@
 			reg = <0x7d508200 0x58>;
 			interrupt-parent = <&bsc_irq>;
 			interrupts = <1>;
-			clock-frequency = <200000>;
+			clock-frequency = <97500>;
 			#address-cells = <1>;
 			#size-cells = <0>;
 			status = "disabled";
@@ -453,7 +453,7 @@
 			reg = <0x7d508280 0x58>;
 			interrupt-parent = <&bsc_irq>;
 			interrupts = <2>;
-			clock-frequency = <200000>;
+			clock-frequency = <97500>;
 			#address-cells = <1>;
 			#size-cells = <0>;
 			status = "disabled";
